gruntfile.js

```diff
@@ -1,4 +1,39 @@
 module.exports = function (grunt) {
+
+    const aiCoreDefaultNameReplacements = [
```

Comment:
These files are read and processed dynamically, so all of the values listed in the source code cause the automatic replacements of any possible values.

The difference between `core` and the rest is that the `core` has the main exported definitions and therefore needs to import from the local file, while the rest import from the module.